docs/recipes/ActionQueue.md

```diff
@@ -0,0 +1,156 @@
+# Action Queue
+
+## Building a simple action queue
+
+Sometimes it might be desirable to dynamically add commands to a queue and
+handle each one in succession. Lets say for example we want to be able to
+drag tasks on a taskboard. Whenevever a drag movement ends we have to send the
+new position of the task to the server as well, so that it persists on a page reload.
+If our request fails we want to roll back and put the task at its source position.
+But what happens if we drag multiple tasks in quick succession?
+Because of their asynchronous nature requests could resolve in any order.
+Meaning we might produce inconsistent states.
+There would be no way for us to control in which order our requests resolve,
+and because each request is triggered by its own action we cannot make use of
+`Cmd.list(cmds, { sequence: true })`.
+Instead we need to build a queue ourselves which luckily is quite easy to do with redux-loop.
+The example below shows, how you could model your initial state as well as your _QueueReducer_.
+
+```js
+const initialState = {
+  queue: [],
+  error: false,
+  //other fields
+};
+
+function handleItemDraggedAction(state, action){
```

Comment:
can you add the main reducer body (with the switch statement) up here? I think it'd be helpful to have the whole thing complete